bobikApp.xcodeproj/project.pbxproj

```diff
@@ -9,23 +9,27 @@
 /* Begin PBXBuildFile section */
 		7F3BA9EE2B5D9F630092C286 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7F3BA9ED2B5D9F630092C286 /* AppDelegate.swift */; };
 		7F3BA9F02B5D9F630092C286 /* SceneDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7F3BA9EF2B5D9F630092C286 /* SceneDelegate.swift */; };
-		7F3BA9F22B5D9F630092C286 /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7F3BA9F12B5D9F630092C286 /* ViewController.swift */; };
```

Comment:
You removed the class from the project, but I see there is still a file in the folder. So you've only removed the file reference from the project.